src/components/Wallet.tsx

```diff
@@ -46,7 +55,7 @@ const Wallet: React.FC<RouteComponentProps> = (props: WalletProps) => {
   if (loadingLabel) {
     content = (
       <>
-        <FontAwesomeIcon icon={faSpinner} />
+        <FontAwesomeIcon icon={faSpinner} spin={!!loadingLabel} />
```

Comment:
I think it's good but the condition is redundant. You can just add:
```js
<FontAwesomeIcon icon={faSpinner} spin />
```

Reply:
That's true. Fixing it